### PR TITLE
Followers Research And Development Addition 

### PR DESCRIPTION
**Pull request by SquishyOranges**
## About The Pull Request

Basically what it says, adds research to the followers. Adds the ability to research and the ability to **ONLY** print researched medical-related items.

## Why It's Good For The Game
Followers do RnD too!

## Changelog
:cl:
add: Fixed the ability for Follower RND to actually be able to create RnD points.
add: Research console, circuit printer, medical lathe
/:cl:

### CODE REVIEW

**Comment by Isy232** [comment]
FYI this lets them print rapid syringe guns and grenade casings.

Not that I mind, if you fuck with the followers perhaps you deserve to get shit on.

**Comment by SquishyOranges** [comment]
> FYI this lets them print rapid syringe guns and grenade casings.
> 
> Not that I mind, if you fuck with the followers perhaps you deserve to get shit on.

You cannot print nonmedical stuffs. So no nades, none of that.

**Comment by FantasticFwoosh** [comment]
Oh, this was partially what I was working on, ill just make some notes and use this PR as a jumping point when it gets merged.

Dont forget you can connect your machines to a Follower server box the same way you can connect a ore silo on a multi-tool buffer, so you can technically negotiate to give Oasis if they source their own kit access to using the follower network (it needs a dedicated cooling room ``@SquishyOranges``)

**Comment by Mannybrado** [comment]
why do they need RnD? they already have everything they need and more. this is powercreep.

**Comment by SquishyOranges** [comment]
> Oh, this was partially what I was working on, ill just make some notes and use this PR as a jumping point when it gets merged.
> 
> Dont forget you can connect your machines to a Follower server box the same way you can connect a ore silo on a multi-tool buffer, so you can technically negotiate to give Oasis if they source their own kit access to using the follower network (it needs a dedicated cooling room ``@SquishyOranges``)

No atmos means it doesn't need a cooling

**Comment by FantasticFwoosh** [comment]
> 
> 
> why do they need RnD? they already have everything they need and more. this is powercreep.

You'll need to look to ``code/controllers/subsystem/research.dm`` on this PR, where followers generate a passive income of barely a tenth of a usual server box. This can be coupled with (which havent been drawn out) prospective follower only nodes to use in the construction of medical equipment.

There are ways to get around this though, like sacrificing blueprints (indiscriminately might raise eyebrows) to directly get a dump of points.

* A streamlined follower tech tree will hopefully drop the requirement for a medical lathe and allow us to explore things followers would probably value in a technological niche. Fast on demand fabrication of chemicals from essential reagents in the machine, granter books & ability to make their own boards to expand the clinics capabilities.

* RP wise a server box is a valuable asset, if you control the follower base you essentially control the server box and can rebuild follower RND and connect via multitool from anywhere in the wasteland. Stealing tech is just like stealing anything from the base, so its a right that fought for or bartered.

File: _maps/map_files/Pahrump/Pahrump-Underground-1.dmm [hunk 1]
```diff
@@ -13460,6 +13463,7 @@
 "kSn" = (
 /obj/structure/frame/machine,
 /obj/structure/window/spawner,
+/obj/item/circuitboard/machine/circuit_imprinter,
```
Comment (DeveloperLoser) [inline _maps/map_files/Pahrump/Pahrump-Underground-1.dmm:13466]: They can just print a proto-lathe, no?

**Comment by ogrigbe** [comment]
no

**Pull request closed without merging**